qiskit/visualization/utils.py

```diff
@@ -372,7 +431,7 @@ def _get_layered_instructions(circuit, reverse_bits=False, justify=None, idle_wi
 
     nodes = [[node for node in layer if any(q in qubits for q in node.qargs)] for layer in nodes]
 
-    return qubits, clbits, nodes
+    return circuit, qubits, clbits, nodes
```

Comment:
Another small thing, not essential now but for next time, I think it's generally better for new args and return values to be added at the end of the line to prevent unintentional errors where it's used elsewhere

Reply:
Got it.